src/events/ldap.js

```diff
@@ -28,6 +28,9 @@ function bindWrapper(bindFunction) {
                 controls = c;
             }
             utils.debugLog(`LDAP.js bind() wrapper - name: ${name}`);
+            if (this.urls.length > 0) {
+                this.url = this.urls[0];
```

Comment:
Will this override `this.url` in some way?

Reply:
this.url is now undefined  because even do the documentation say you can set a string or array when
`const client = ldap.createClient({
        url: 'ldap://localhost:4389',
    });`
    
  the library set the .urls property while .url stay undefined and so we need to set it to avoid exceptions later on in your code.
  
I think this code is not 100% correct because the user could setup multiple servers for redundancy and so like this we pick only the first while maybe in case of errors we should try to pick other server in the array but at least in this way I did not change the logic.